docker-bake.hcl

```
@@ -161,6 +161,9 @@ variable CONTENT_BUILD_MATRIX {
 
       # R-4.4, Python-3.12.4, Quarto-1.4.557 (polyfill.js vulnerability patch)
       {os = "ubuntu2204", os_alt = "jammy", r = "4.4.1", py = "3.12.4", drivers = "2024.03.0", quarto = "1.4.557"},
+
+      # R-4.5, Python-3.13, Quarto-1.7
+      {os = "ubuntu2204", os_alt = "jammy", r = "4.5.1", py = "3.13.5", drivers = "2025.03.0", quarto = "1.7.32"},
```

Let's keep this consistent with the existing content images for now. We can upgrade them all in a separate PR.
```suggestion
      {os = "ubuntu2204", os_alt = "jammy", r = "4.5.1", py = "3.13.5", drivers = "2024.03.0", quarto = "1.7.32"},
```

---

Done.